### DISCUSSION TITLE
Suggestions Sought: Attractive Mobile-friendly Resource List

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1581] [Support]
Looking for implementable ideas for making our [Community Resources page](https://sareview.org/resources)  more accessible (and mobile-friendly).

Ideas?

**Comment by gavyukko-creator** [comment]
Bardzo ciekawa dyskusja na temat rozwoju PubPub i przyszłości otwartych platform publikacyjnych. Szczególnie ważne wydaje się podejście do współpracy społeczności, przejrzystości procesów oraz tworzenia narzędzi, które ułatwiają dzielenie się wiedzą. Projekty open source takie jak ten pokazują, że zaangażowanie użytkowników może realnie wpływać na rozwój technologii i jej praktyczne zastosowania. Warto również obserwować, jak różne społeczności internetowe wykorzystują podobne rozwiązania do budowania wartościowych zasobów i wymiany doświadczeń, np. w innych obszarach cyfrowych, takich jak [spin mama](https://spinmama.pl/). Mam nadzieję, że PubPub będzie dalej rozwijany w kierunku jeszcze większej dostępności, elastyczności i współpracy między użytkownikami.

**Reply by jamewills420-stack** [reply]
Great suggestions for improving community resource pages. Making information easier to find on mobile devices can help more people access important services quickly. Including organized sections such as [Benton Court Locator](https://bentoncountycourt.org) can also help users find relevant public information with less effort. Clear layouts and simple navigation are valuable improvements for any resource page.
